Template10 (Library)/Controls/HamburgerButtonInfo.cs

```diff
@@ -83,6 +79,14 @@ public bool IsEnabled
             DependencyProperty.Register(nameof(IsEnabled), typeof(bool),
                 typeof(HamburgerButtonInfo), new PropertyMetadata(true));
 
+        public string ToolTipText
```

Comment:
First, please note that ToolTip is a ContentControl and not a Text control. This means developers can choose to put XAML controls into it, not just a string. Creating a ToolTipText property instead of using type `object` means you limit developer options. We don't want to do that.